mlibrary/my_account
# [LIBSEARCH-903] ACCOUNT: Swap 404 image for the "airplane and searchlight" one from website and blogs

# Overview
The stack of books used on the original 404 page in Account is also used on the “you have no items checked out” page, which can cause user confusion. To remove that confusion, and for consistency across Library sites, the image has been replaced with the iconic spotlight image.

This pull request resolves [LIBSEARCH-903](https://mlit.atlassian.net/browse/LIBSEARCH-903).

This pull request also removes the warning about SMS message not being available because SMS messages are working again. 

This pull request also updates `init.sh`, `pre-commit`, and the `README` to use the newer `docker compose` instead of `docker-compose`. 

## Testing
List instructions on how to test the pull request. Some examples:

- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
  - Break the new/updated unit tests to make sure they're working properly.
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Go to a 404 page and see if the illustration is now the new one. Go to a page where there are no checkouts, and see if it still shows the original illustration.
